assets/styles/index.scss

```diff
@@ -3460,16 +3460,23 @@ a.wire-articles__versions {
         padding: 0;
         margin: 10px 0;
     }
+
```

Comment:
@fritzSF I had to move these out, as the design that the customer provided had colours inside the dot points and not just in the block text

Reply:
OK, no problem